Base/usr/share/man/man2/geteuid.md

```diff
@@ -0,0 +1,25 @@
+## Name
+
+geteuid, getegid - get effective user / group id
+
+## Synopsis
+
+```**c++
+#include <unistd.h>
+
+uid_t geteuid(void);
+gid_t getegid(void);
+```
+
+## Description
+
+Returns the effective user or group id.
+
+## See also
+
+* [`setuid_overview`(7)](../man7/setuid_overview.md)
```

Comment:
No need for "overview", it should be just `setuid(7)`

Reply:
I claim that most people don't have the category numbers for man pages memorized :)